test/unittests/t_util.cc

```diff
@@ -1763,3 +1764,57 @@ TEST_F(T_Util, DiffTree) {
   EXPECT_TRUE(DiffTree("./.", "."));
   EXPECT_FALSE(DiffTree(".", "/"));
 }
+
+TEST(BoundaryValueTest, 2Bin_Empty) {
+  Log2Hist log2hist(2);
+  log2hist.Add(10);
+  log2hist.Add(11);
+  log2hist.Add(11.5);
+
+  atomic_int32 *bins = log2hist.GetBins();
+  uint res[3] = {3, 0, 0};
+  for (int i = 0; i < 3; i++)
+  {
+    EXPECT_EQ(res[i], atomic_read32(&bins[i]));
+  }
+}
+
+TEST(BoundaryValueTest, 2Bins) {
```

Comment:
Let's rename to `TEST(Log2Histogram, 2Bins)`

Reply:
Done!